docs/tooling.md

```diff
@@ -107,10 +120,3 @@ If you're still having issues, please ensure that the channels in the path and t
 >
 > This does not work for web builds.
 > </details>
-
-
-## Other templates
-
-There are many other Bevy templates out there.
-Check out the [templates category](https://bevyengine.org/assets/#templates) on Bevy Assets for more options.
-Even if you don't end up using them, they are a great way to learn how to implement certain features you might be interested in.
```

Comment:
Last time around, people explicitly asked us to include this somewhere IIRC
Not that I think it's useful, but just saying